nexus/types/src/deployment.rs

```diff
@@ -1065,6 +983,7 @@ impl BlueprintDatasetDisposition {
 /// Information about a dataset as recorded in a blueprint
 #[derive(Debug, Clone, Eq, PartialEq, JsonSchema, Deserialize, Serialize)]
 pub struct BlueprintDatasetConfig {
+    // TODO: Display this in diffs - leave for now, for backwards compat
```

Comment:
I'm trying to change as little of the diff output as possible with this change to demonstrate correctness.

In a follow up PR I'll add a column for disposition.